___Execucao_com 2 Treads___:
CarbonTracker: The following components were found: CPU with device(s) cpu:1, cpu:0.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

Initialization:
unew_norm = 0.0509645
u_norm = 0.705265
error = 0.705287
-------------------------------

Converged after 18500 iterations
unew_norm: 0.412466
diff: 9.97582e-06
tolerance: 1e-05
error: 0.532394
Execution Time: 3.53741
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Average carbon intensity during training was 475.00 gCO2/kWh at detected location: UNDETECTED.
CarbonTracker: 
Actual consumption for 1 epoch(s):
	Time:	0:00:04
	Energy:	0.0001 kWh
	CO2eq:	0.0581 g
	This is equivalent to:
	0.0005 km travelled by car
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Live carbon intensity could not be fetched at detected location: Unknown. Defaulted to average carbon intensity for world in 2019 of 475.00 gCO2/kWh. at detected location: UNDETECTED.
CarbonTracker: 
Predicted consumption for 1 epoch(s):
	Time:	0:00:04
	Energy:	0.0001 kWh
	CO2eq:	0.0581 g
	This is equivalent to:
	0.0005 km travelled by car
CarbonTracker: Finished monitoring.
___Execucao_de 2 Treads_concluida___
___Execucao_com 4 Treads___:
CarbonTracker: The following components were found: CPU with device(s) cpu:1, cpu:0.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

Initialization:
unew_norm = 0.0509645
u_norm = 0.705265
error = 0.705287
-------------------------------

Converged after 18500 iterations
unew_norm: 0.412466
diff: 9.97582e-06
tolerance: 1e-05
error: 0.532394
Execution Time: 1.89733
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Average carbon intensity during training was 475.00 gCO2/kWh at detected location: UNDETECTED.
CarbonTracker: 
Actual consumption for 1 epoch(s):
	Time:	0:00:02
	Energy:	0.0001 kWh
	CO2eq:	0.0450 g
	This is equivalent to:
	0.0004 km travelled by car
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Live carbon intensity could not be fetched at detected location: Unknown. Defaulted to average carbon intensity for world in 2019 of 475.00 gCO2/kWh. at detected location: UNDETECTED.
CarbonTracker: 
Predicted consumption for 1 epoch(s):
	Time:	0:00:02
	Energy:	0.0001 kWh
	CO2eq:	0.0450 g
	This is equivalent to:
	0.0004 km travelled by car
CarbonTracker: Finished monitoring.
___Execucao_de 4 Treads_concluida___
___Execucao_com 6 Treads___:
CarbonTracker: The following components were found: CPU with device(s) cpu:1, cpu:0.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

Initialization:
unew_norm = 0.0509645
u_norm = 0.705265
error = 0.705287
-------------------------------

Converged after 18500 iterations
unew_norm: 0.412466
diff: 9.97582e-06
tolerance: 1e-05
error: 0.532394
Execution Time: 1.36197
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Average carbon intensity during training was 475.00 gCO2/kWh at detected location: UNDETECTED.
CarbonTracker: 
Actual consumption for 1 epoch(s):
	Time:	0:00:01
	Energy:	0.0001 kWh
	CO2eq:	0.0342 g
	This is equivalent to:
	0.0003 km travelled by car
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Live carbon intensity could not be fetched at detected location: Unknown. Defaulted to average carbon intensity for world in 2019 of 475.00 gCO2/kWh. at detected location: UNDETECTED.
CarbonTracker: 
Predicted consumption for 1 epoch(s):
	Time:	0:00:01
	Energy:	0.0001 kWh
	CO2eq:	0.0342 g
	This is equivalent to:
	0.0003 km travelled by car
CarbonTracker: Finished monitoring.
___Execucao_de 6 Treads_concluida___
___Execucao_com 8 Treads___:
CarbonTracker: The following components were found: CPU with device(s) cpu:1, cpu:0.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

Initialization:
unew_norm = 0.0509645
u_norm = 0.705265
error = 0.705287
-------------------------------

Converged after 18500 iterations
unew_norm: 0.412466
diff: 9.97582e-06
tolerance: 1e-05
error: 0.532394
Execution Time: 1.05540
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Average carbon intensity during training was 475.00 gCO2/kWh at detected location: UNDETECTED.
CarbonTracker: 
Actual consumption for 1 epoch(s):
	Time:	0:00:01
	Energy:	0.0001 kWh
	CO2eq:	0.0289 g
	This is equivalent to:
	0.0003 km travelled by car
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Live carbon intensity could not be fetched at detected location: Unknown. Defaulted to average carbon intensity for world in 2019 of 475.00 gCO2/kWh. at detected location: UNDETECTED.
CarbonTracker: 
Predicted consumption for 1 epoch(s):
	Time:	0:00:01
	Energy:	0.0001 kWh
	CO2eq:	0.0289 g
	This is equivalent to:
	0.0003 km travelled by car
CarbonTracker: Finished monitoring.
___Execucao_de 8 Treads_concluida___
___Execucao_com 10 Treads___:
CarbonTracker: The following components were found: CPU with device(s) cpu:1, cpu:0.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

Initialization:
unew_norm = 0.0509645
u_norm = 0.705265
error = 0.705287
-------------------------------

Converged after 18500 iterations
unew_norm: 0.412466
diff: 9.97582e-06
tolerance: 1e-05
error: 0.532394
Execution Time: 0.84133
CarbonTracker: WARNING - Epoch duration is too short for a measurement to be collected.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Average carbon intensity during training was 475.00 gCO2/kWh at detected location: UNDETECTED.
CarbonTracker: 
Actual consumption for 1 epoch(s):
	Time:	0:00:01
	Energy:	0.0000 kWh
	CO2eq:	0.0000 g
	This is equivalent to:
	0.0000 km travelled by car
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Live carbon intensity could not be fetched at detected location: Unknown. Defaulted to average carbon intensity for world in 2019 of 475.00 gCO2/kWh. at detected location: UNDETECTED.
CarbonTracker: 
Predicted consumption for 1 epoch(s):
	Time:	0:00:01
	Energy:	0.0000 kWh
	CO2eq:	0.0211 g
	This is equivalent to:
	0.0002 km travelled by car
CarbonTracker: Finished monitoring.
___Execucao_de 10 Treads_concluida___
___Execucao_com 12 Treads___:
CarbonTracker: The following components were found: CPU with device(s) cpu:1, cpu:0.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

Initialization:
unew_norm = 0.0509645
u_norm = 0.705265
error = 0.705287
-------------------------------

Converged after 18500 iterations
unew_norm: 0.412466
diff: 9.97582e-06
tolerance: 1e-05
error: 0.532394
Execution Time: 0.86662
CarbonTracker: WARNING - Epoch duration is too short for a measurement to be collected.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Average carbon intensity during training was 475.00 gCO2/kWh at detected location: UNDETECTED.
CarbonTracker: 
Actual consumption for 1 epoch(s):
	Time:	0:00:01
	Energy:	0.0000 kWh
	CO2eq:	0.0000 g
	This is equivalent to:
	0.0000 km travelled by car
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Live carbon intensity could not be fetched at detected location: Unknown. Defaulted to average carbon intensity for world in 2019 of 475.00 gCO2/kWh. at detected location: UNDETECTED.
CarbonTracker: 
Predicted consumption for 1 epoch(s):
	Time:	0:00:01
	Energy:	0.0000 kWh
	CO2eq:	0.0230 g
	This is equivalent to:
	0.0002 km travelled by car
CarbonTracker: Finished monitoring.
___Execucao_de 12 Treads_concluida___
___Execucao_com 14 Treads___:
CarbonTracker: The following components were found: CPU with device(s) cpu:1, cpu:0.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

Initialization:
unew_norm = 0.0509645
u_norm = 0.705265
error = 0.705287
-------------------------------

Converged after 18500 iterations
unew_norm: 0.412466
diff: 9.97582e-06
tolerance: 1e-05
error: 0.532394
Execution Time: 0.64368
CarbonTracker: WARNING - Epoch duration is too short for a measurement to be collected.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Average carbon intensity during training was 475.00 gCO2/kWh at detected location: UNDETECTED.
CarbonTracker: 
Actual consumption for 1 epoch(s):
	Time:	0:00:01
	Energy:	0.0000 kWh
	CO2eq:	0.0000 g
	This is equivalent to:
	0.0000 km travelled by car
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Live carbon intensity could not be fetched at detected location: Unknown. Defaulted to average carbon intensity for world in 2019 of 475.00 gCO2/kWh. at detected location: UNDETECTED.
CarbonTracker: 
Predicted consumption for 1 epoch(s):
	Time:	0:00:01
	Energy:	0.0000 kWh
	CO2eq:	0.0000 g
	This is equivalent to:
	0.0000 km travelled by car
CarbonTracker: Finished monitoring.
___Execucao_de 14 Treads_concluida___
___Execucao_com 16 Treads___:
CarbonTracker: The following components were found: CPU with device(s) cpu:1, cpu:0.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

Initialization:
unew_norm = 0.0509645
u_norm = 0.705265
error = 0.705287
-------------------------------

Converged after 18500 iterations
unew_norm: 0.412466
diff: 9.97582e-06
tolerance: 1e-05
error: 0.532394
Execution Time: 0.61125
CarbonTracker: WARNING - Epoch duration is too short for a measurement to be collected.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Average carbon intensity during training was 475.00 gCO2/kWh at detected location: UNDETECTED.
CarbonTracker: 
Actual consumption for 1 epoch(s):
	Time:	0:00:01
	Energy:	0.0000 kWh
	CO2eq:	0.0000 g
	This is equivalent to:
	0.0000 km travelled by car
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Live carbon intensity could not be fetched at detected location: Unknown. Defaulted to average carbon intensity for world in 2019 of 475.00 gCO2/kWh. at detected location: UNDETECTED.
CarbonTracker: 
Predicted consumption for 1 epoch(s):
	Time:	0:00:01
	Energy:	0.0000 kWh
	CO2eq:	0.0000 g
	This is equivalent to:
	0.0000 km travelled by car
CarbonTracker: Finished monitoring.
___Execucao_de 16 Treads_concluida___
___Execucao_com 18 Treads___:
CarbonTracker: The following components were found: CPU with device(s) cpu:1, cpu:0.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

Initialization:
unew_norm = 0.0509645
u_norm = 0.705265
error = 0.705287
-------------------------------

Converged after 18500 iterations
unew_norm: 0.412466
diff: 9.97582e-06
tolerance: 1e-05
error: 0.532394
Execution Time: 0.64562
CarbonTracker: WARNING - Epoch duration is too short for a measurement to be collected.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Average carbon intensity during training was 475.00 gCO2/kWh at detected location: UNDETECTED.
CarbonTracker: 
Actual consumption for 1 epoch(s):
	Time:	0:00:01
	Energy:	0.0000 kWh
	CO2eq:	0.0000 g
	This is equivalent to:
	0.0000 km travelled by car
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Live carbon intensity could not be fetched at detected location: Unknown. Defaulted to average carbon intensity for world in 2019 of 475.00 gCO2/kWh. at detected location: UNDETECTED.
CarbonTracker: 
Predicted consumption for 1 epoch(s):
	Time:	0:00:01
	Energy:	0.0000 kWh
	CO2eq:	0.0000 g
	This is equivalent to:
	0.0000 km travelled by car
CarbonTracker: Finished monitoring.
___Execucao_de 18 Treads_concluida___
___Execucao_com 20 Treads___:
CarbonTracker: The following components were found: CPU with device(s) cpu:1, cpu:0.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

Initialization:
unew_norm = 0.0509645
u_norm = 0.705265
error = 0.705287
-------------------------------

Converged after 18500 iterations
unew_norm: 0.412466
diff: 9.97582e-06
tolerance: 1e-05
error: 0.532394
Execution Time: 0.48698
CarbonTracker: WARNING - Epoch duration is too short for a measurement to be collected.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Average carbon intensity during training was 475.00 gCO2/kWh at detected location: UNDETECTED.
CarbonTracker: 
Actual consumption for 1 epoch(s):
	Time:	0:00:01
	Energy:	0.0000 kWh
	CO2eq:	0.0000 g
	This is equivalent to:
	0.0000 km travelled by car
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Live carbon intensity could not be fetched at detected location: Unknown. Defaulted to average carbon intensity for world in 2019 of 475.00 gCO2/kWh. at detected location: UNDETECTED.
CarbonTracker: 
Predicted consumption for 1 epoch(s):
	Time:	0:00:01
	Energy:	0.0000 kWh
	CO2eq:	0.0000 g
	This is equivalent to:
	0.0000 km travelled by car
CarbonTracker: Finished monitoring.
___Execucao_de 20 Treads_concluida___
___Execucao_com 22 Treads___:
CarbonTracker: The following components were found: CPU with device(s) cpu:1, cpu:0.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

Initialization:
unew_norm = 0.0509645
u_norm = 0.705265
error = 0.705287
-------------------------------

Converged after 18500 iterations
unew_norm: 0.412466
diff: 9.97582e-06
tolerance: 1e-05
error: 0.532394
Execution Time: 0.79799
CarbonTracker: WARNING - Epoch duration is too short for a measurement to be collected.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Average carbon intensity during training was 475.00 gCO2/kWh at detected location: UNDETECTED.
CarbonTracker: 
Actual consumption for 1 epoch(s):
	Time:	0:00:01
	Energy:	0.0000 kWh
	CO2eq:	0.0000 g
	This is equivalent to:
	0.0000 km travelled by car
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Live carbon intensity could not be fetched at detected location: Unknown. Defaulted to average carbon intensity for world in 2019 of 475.00 gCO2/kWh. at detected location: UNDETECTED.
CarbonTracker: 
Predicted consumption for 1 epoch(s):
	Time:	0:00:01
	Energy:	0.0000 kWh
	CO2eq:	0.0000 g
	This is equivalent to:
	0.0000 km travelled by car
CarbonTracker: Finished monitoring.
___Execucao_de 22 Treads_concluida___
___Execucao_com 24 Treads___:
CarbonTracker: The following components were found: CPU with device(s) cpu:1, cpu:0.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

Initialization:
unew_norm = 0.0509645
u_norm = 0.705265
error = 0.705287
-------------------------------

Converged after 18500 iterations
unew_norm: 0.412466
diff: 9.97582e-06
tolerance: 1e-05
error: 0.532394
Execution Time: 0.75606
CarbonTracker: WARNING - Epoch duration is too short for a measurement to be collected.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Average carbon intensity during training was 475.00 gCO2/kWh at detected location: UNDETECTED.
CarbonTracker: 
Actual consumption for 1 epoch(s):
	Time:	0:00:01
	Energy:	0.0000 kWh
	CO2eq:	0.0000 g
	This is equivalent to:
	0.0000 km travelled by car
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Live carbon intensity could not be fetched at detected location: Unknown. Defaulted to average carbon intensity for world in 2019 of 475.00 gCO2/kWh. at detected location: UNDETECTED.
CarbonTracker: 
Predicted consumption for 1 epoch(s):
	Time:	0:00:01
	Energy:	0.0000 kWh
	CO2eq:	0.0000 g
	This is equivalent to:
	0.0000 km travelled by car
CarbonTracker: Finished monitoring.
___Execucao_de 24 Treads_concluida___
___Execucao_com 26 Treads___:
CarbonTracker: The following components were found: CPU with device(s) cpu:1, cpu:0.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

Initialization:
unew_norm = 0.0509645
u_norm = 0.705265
error = 0.705287
-------------------------------

Converged after 18500 iterations
unew_norm: 0.412466
diff: 9.97582e-06
tolerance: 1e-05
error: 0.532394
Execution Time: 0.72931
CarbonTracker: WARNING - Epoch duration is too short for a measurement to be collected.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Average carbon intensity during training was 475.00 gCO2/kWh at detected location: UNDETECTED.
CarbonTracker: 
Actual consumption for 1 epoch(s):
	Time:	0:00:01
	Energy:	0.0000 kWh
	CO2eq:	0.0000 g
	This is equivalent to:
	0.0000 km travelled by car
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Live carbon intensity could not be fetched at detected location: Unknown. Defaulted to average carbon intensity for world in 2019 of 475.00 gCO2/kWh. at detected location: UNDETECTED.
CarbonTracker: 
Predicted consumption for 1 epoch(s):
	Time:	0:00:01
	Energy:	0.0000 kWh
	CO2eq:	0.0000 g
	This is equivalent to:
	0.0000 km travelled by car
CarbonTracker: Finished monitoring.
___Execucao_de 26 Treads_concluida___
___Execucao_com 28 Treads___:
CarbonTracker: The following components were found: CPU with device(s) cpu:1, cpu:0.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

Initialization:
unew_norm = 0.0509645
u_norm = 0.705265
error = 0.705287
-------------------------------

Converged after 18500 iterations
unew_norm: 0.412466
diff: 9.97582e-06
tolerance: 1e-05
error: 0.532394
Execution Time: 0.73380
CarbonTracker: WARNING - Epoch duration is too short for a measurement to be collected.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Average carbon intensity during training was 475.00 gCO2/kWh at detected location: UNDETECTED.
CarbonTracker: 
Actual consumption for 1 epoch(s):
	Time:	0:00:01
	Energy:	0.0000 kWh
	CO2eq:	0.0000 g
	This is equivalent to:
	0.0000 km travelled by car
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Live carbon intensity could not be fetched at detected location: Unknown. Defaulted to average carbon intensity for world in 2019 of 475.00 gCO2/kWh. at detected location: UNDETECTED.
CarbonTracker: 
Predicted consumption for 1 epoch(s):
	Time:	0:00:01
	Energy:	0.0000 kWh
	CO2eq:	0.0000 g
	This is equivalent to:
	0.0000 km travelled by car
CarbonTracker: Finished monitoring.
___Execucao_de 28 Treads_concluida___
___Execucao_com 30 Treads___:
CarbonTracker: The following components were found: CPU with device(s) cpu:1, cpu:0.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

Initialization:
unew_norm = 0.0509645
u_norm = 0.705265
error = 0.705287
-------------------------------

Converged after 18500 iterations
unew_norm: 0.412466
diff: 9.97582e-06
tolerance: 1e-05
error: 0.532394
Execution Time: 0.68682
CarbonTracker: WARNING - Epoch duration is too short for a measurement to be collected.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Average carbon intensity during training was 475.00 gCO2/kWh at detected location: UNDETECTED.
CarbonTracker: 
Actual consumption for 1 epoch(s):
	Time:	0:00:01
	Energy:	0.0000 kWh
	CO2eq:	0.0000 g
	This is equivalent to:
	0.0000 km travelled by car
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Live carbon intensity could not be fetched at detected location: Unknown. Defaulted to average carbon intensity for world in 2019 of 475.00 gCO2/kWh. at detected location: UNDETECTED.
CarbonTracker: 
Predicted consumption for 1 epoch(s):
	Time:	0:00:01
	Energy:	0.0000 kWh
	CO2eq:	0.0000 g
	This is equivalent to:
	0.0000 km travelled by car
CarbonTracker: Finished monitoring.
___Execucao_de 30 Treads_concluida___
___Execucao_com 32 Treads___:
CarbonTracker: The following components were found: CPU with device(s) cpu:1, cpu:0.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

Initialization:
unew_norm = 0.0509645
u_norm = 0.705265
error = 0.705287
-------------------------------

Converged after 18500 iterations
unew_norm: 0.412466
diff: 9.97582e-06
tolerance: 1e-05
error: 0.532394
Execution Time: 0.65333
CarbonTracker: WARNING - Epoch duration is too short for a measurement to be collected.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Average carbon intensity during training was 475.00 gCO2/kWh at detected location: UNDETECTED.
CarbonTracker: 
Actual consumption for 1 epoch(s):
	Time:	0:00:01
	Energy:	0.0000 kWh
	CO2eq:	0.0000 g
	This is equivalent to:
	0.0000 km travelled by car
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Live carbon intensity could not be fetched at detected location: Unknown. Defaulted to average carbon intensity for world in 2019 of 475.00 gCO2/kWh. at detected location: UNDETECTED.
CarbonTracker: 
Predicted consumption for 1 epoch(s):
	Time:	0:00:01
	Energy:	0.0000 kWh
	CO2eq:	0.0000 g
	This is equivalent to:
	0.0000 km travelled by car
CarbonTracker: Finished monitoring.
___Execucao_de 32 Treads_concluida___
___Execucao_com 34 Treads___:
CarbonTracker: The following components were found: CPU with device(s) cpu:1, cpu:0.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

Initialization:
unew_norm = 0.0509645
u_norm = 0.705265
error = 0.705287
-------------------------------

Converged after 18500 iterations
unew_norm: 0.412466
diff: 9.97582e-06
tolerance: 1e-05
error: 0.532394
Execution Time: 0.61458
CarbonTracker: WARNING - Epoch duration is too short for a measurement to be collected.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Average carbon intensity during training was 475.00 gCO2/kWh at detected location: UNDETECTED.
CarbonTracker: 
Actual consumption for 1 epoch(s):
	Time:	0:00:01
	Energy:	0.0000 kWh
	CO2eq:	0.0000 g
	This is equivalent to:
	0.0000 km travelled by car
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Live carbon intensity could not be fetched at detected location: Unknown. Defaulted to average carbon intensity for world in 2019 of 475.00 gCO2/kWh. at detected location: UNDETECTED.
CarbonTracker: 
Predicted consumption for 1 epoch(s):
	Time:	0:00:01
	Energy:	0.0000 kWh
	CO2eq:	0.0000 g
	This is equivalent to:
	0.0000 km travelled by car
CarbonTracker: Finished monitoring.
___Execucao_de 34 Treads_concluida___
___Execucao_com 36 Treads___:
CarbonTracker: The following components were found: CPU with device(s) cpu:1, cpu:0.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

Initialization:
unew_norm = 0.0509645
u_norm = 0.705265
error = 0.705287
-------------------------------

Converged after 18500 iterations
unew_norm: 0.412466
diff: 9.97582e-06
tolerance: 1e-05
error: 0.532394
Execution Time: 0.64072
CarbonTracker: WARNING - Epoch duration is too short for a measurement to be collected.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Average carbon intensity during training was 475.00 gCO2/kWh at detected location: UNDETECTED.
CarbonTracker: 
Actual consumption for 1 epoch(s):
	Time:	0:00:01
	Energy:	0.0000 kWh
	CO2eq:	0.0000 g
	This is equivalent to:
	0.0000 km travelled by car
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Live carbon intensity could not be fetched at detected location: Unknown. Defaulted to average carbon intensity for world in 2019 of 475.00 gCO2/kWh. at detected location: UNDETECTED.
CarbonTracker: 
Predicted consumption for 1 epoch(s):
	Time:	0:00:01
	Energy:	0.0000 kWh
	CO2eq:	0.0000 g
	This is equivalent to:
	0.0000 km travelled by car
CarbonTracker: Finished monitoring.
___Execucao_de 36 Treads_concluida___
___Execucao_com 38 Treads___:
CarbonTracker: The following components were found: CPU with device(s) cpu:1, cpu:0.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

Initialization:
unew_norm = 0.0509645
u_norm = 0.705265
error = 0.705287
-------------------------------

Converged after 18500 iterations
unew_norm: 0.412466
diff: 9.97582e-06
tolerance: 1e-05
error: 0.532394
Execution Time: 0.61225
CarbonTracker: WARNING - Epoch duration is too short for a measurement to be collected.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Average carbon intensity during training was 475.00 gCO2/kWh at detected location: UNDETECTED.
CarbonTracker: 
Actual consumption for 1 epoch(s):
	Time:	0:00:01
	Energy:	0.0000 kWh
	CO2eq:	0.0000 g
	This is equivalent to:
	0.0000 km travelled by car
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Live carbon intensity could not be fetched at detected location: Unknown. Defaulted to average carbon intensity for world in 2019 of 475.00 gCO2/kWh. at detected location: UNDETECTED.
CarbonTracker: 
Predicted consumption for 1 epoch(s):
	Time:	0:00:01
	Energy:	0.0000 kWh
	CO2eq:	0.0000 g
	This is equivalent to:
	0.0000 km travelled by car
CarbonTracker: Finished monitoring.
___Execucao_de 38 Treads_concluida___
___Execucao_com 40 Treads___:
CarbonTracker: The following components were found: CPU with device(s) cpu:1, cpu:0.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

Initialization:
unew_norm = 0.0509645
u_norm = 0.705265
error = 0.705287
-------------------------------

Converged after 18500 iterations
unew_norm: 0.412466
diff: 9.97582e-06
tolerance: 1e-05
error: 0.532394
Execution Time: 0.61014
CarbonTracker: WARNING - Epoch duration is too short for a measurement to be collected.
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Average carbon intensity during training was 475.00 gCO2/kWh at detected location: UNDETECTED.
CarbonTracker: 
Actual consumption for 1 epoch(s):
	Time:	0:00:01
	Energy:	0.0000 kWh
	CO2eq:	0.0000 g
	This is equivalent to:
	0.0000 km travelled by car
CarbonTracker: INFO - Traceback (most recent call last):
  File "/home/users/acsilveira/.local/lib/python3.11/site-packages/carbontracker/emissions/intensity/intensity.py", line 94, in carbon_intensity
    raise exceptions.IPLocationError("Failed to retrieve location based on IP.")
carbontracker.exceptions.IPLocationError: Failed to retrieve location based on IP.

CarbonTracker: Live carbon intensity could not be fetched at detected location: Unknown. Defaulted to average carbon intensity for world in 2019 of 475.00 gCO2/kWh. at detected location: UNDETECTED.
CarbonTracker: 
Predicted consumption for 1 epoch(s):
	Time:	0:00:01
	Energy:	0.0000 kWh
	CO2eq:	0.0000 g
	This is equivalent to:
	0.0000 km travelled by car
CarbonTracker: Finished monitoring.
___Execucao_de 40 Treads_concluida___
Os resultados foram salvos em log_result_1.txt
Os resultados foram salvos em log_result_2.txt
Os resultados foram salvos em log_result_3.txt
Os resultados foram salvos em log_result_4.txt
Os resultados foram salvos em log_result_5.txt
Os resultados foram salvos em log_result_6.txt
Os resultados foram salvos em log_result_7.txt
Os resultados foram salvos em log_result_8.txt
Os resultados foram salvos em log_result_9.txt
Os resultados foram salvos em log_result_10.txt
